browser/ui/views/frame/brave_browser_view.cc

```diff
@@ -215,6 +217,28 @@ BraveBrowserView::BraveBrowserView(std::unique_ptr<Browser> browser)
   }
 }
 
+void BraveBrowserView::AddedToWidget() {
+  BrowserView::AddedToWidget();
+  // we must call all new views once BraveBrowserView is added to widget
+  fade_animator_ = std::make_unique<views::WidgetFadeAnimator>(frame_);
+  fade_animator_->set_fade_out_duration(base::Milliseconds(1000));
+  pulsing_block_view_ =
+      AddChildView(std::make_unique<views::PulsingBlockView>(browser()));
+  // the position here should be an object to anchor on and responsive when the
+  // browser resizes
+  pulsing_block_view_->SetPosition({750, 26});
```

Comment:
You might want to use ui::ElementTracker like IPH buble does. Also, I think you can get inspired by things in //chrome/browser/ui/user_education/

https://source.chromium.org/chromium/chromium/src/+/main:chrome/browser/ui/user_education/reopen_tab_in_product_help_trigger.h

Reply:
This is sweet find! I'm using element tracker to track shield's icon position. There's also an Interaction sequence interface which can be useful to create complete walkthroughs